sql/telemetry_derived/experiment_enrollment_aggregates_live/view.sql

```diff
@@ -0,0 +1,91 @@
+CREATE
+OR REPLACE VIEW `moz-fx-data-shared-prod`.telemetry_derived.experiment_enrollment_aggregates_live AS (
+  WITH live AS (
+    SELECT
+      event_object AS `type`,
+      event_string_value AS experiment,
+      `moz-fx-data-shared-prod`.udf.get_key(event_map_values, 'branch') AS branch,
+      TIMESTAMP_ADD(
+        TIMESTAMP_TRUNC(`timestamp`, HOUR),
+        INTERVAL (
+          CAST(
+            EXTRACT(
+              MINUTE
+              FROM
+                `timestamp`
+            ) / 5 AS int64
+          ) * 5
+        ) MINUTE
+      ) AS window_start,
```

Comment:
I think this needs a comment indicating that it is truncating timestamp to 5 minute granularity.